### DISCUSSION TITLE
Repo has grown by 60k stars in a Single Day

### DISCUSSION THREAD

**Discussion by creativecreatorormaybenot** [ant-design/ant-design#30537] [General]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
http://www.gitstarhistory.com/#ant-design/ant-design

### Steps to reproduce
none

### What is expected?
The repo should be at 8000 stars today if it had grown normally.

![image](https://user-images.githubusercontent.com/19204050/118386157-daafa280-b604-11eb-91d8-9b06bad32e23.png)

### What is actually happening?
It is actually at 70000 stars.

![Screenshot_2021-05-16 GitHub Tracker ( github_tracker) Twitter](https://user-images.githubusercontent.com/19204050/118386160-e13e1a00-b604-11eb-80a6-5961eee080f5.png)

| Environment | Info |
|---|---|
| antd | undefined |
| React | none |
| System | none |
| Browser | none |

**Comment by creativecreatorormaybenot** [comment]
@afc163 since you closed the issue without a comment, do you have some input on how this came to be?

**Reply by yoyo837** [reply]
https://twitter.com/AntDesignUI/status/1361150625505927168

**Reply by creativecreatorormaybenot** [reply]
Thanks.

**Reply by yoyo837** [reply]
Many stars were lost in that incident, which may be related with this.

**Reply by itayganor** [reply]
@yoyo837 I don't think so,
http://web.archive.org/web/20210426070653/https://github.com/ant-design/ant-design
On April 26th the repository had 6.2k stars. Regardless of the issue they had - where have more than 60k stars came from?

**Reply by yoyo837** [reply]
http://web.archive.org/web/20210105215542/https://github.com/ant-design/ant-design

**Reply by yoyo837** [reply]
Almost all stars were lost in February @itayganor

**Reply by itayganor** [reply]
> Almost all stars were lost in February @itayganor 

Haven't noticed it, thanks for letting me know!

**Reply by yoyo837** [reply]
GITHUB may be doing data recovery.
